ext/tests/span_attribute/function_params.phpt

```diff
@@ -0,0 +1,55 @@
+--TEST--
+Check if function params can be passed via SpanAttribute
+--EXTENSIONS--
+opentelemetry
+--FILE--
+<?php
+namespace OpenTelemetry\API\Instrumentation;
+
+use OpenTelemetry\Instrumentation\WithSpan;
+use OpenTelemetry\Instrumentation\SpanAttribute;
+
+class WithSpanHandler
+{
+    public static function pre(): void
+    {
+        var_dump('pre');
+        var_dump(func_get_args()[7]);
+    }
+    public static function post(): void
+    {
+        var_dump('post');
+    }
+}
+
+#[WithSpan]
+function foo(
+    #[SpanAttribute('renamed_one')] string $one,
+    #[SpanAttribute] int $two,
+    #[SpanAttribute('renamed_three')] float $three,
+    #[SpanAttribute] bool $four,
+    string $five,
+    #[SpanAttribute] string $six
+): void
```

Comment:
Perhaps there should also be a test to see what happens if null, array, object or callable are passed for paramters with `SpanAttribute`.

Reply:
added a test for extra types

Reply:
added a warning message